packages/e2e-cypress/src/no-typescript/test/cypress/integration/home.spec.js

```diff
@@ -0,0 +1,37 @@
+/// <reference path="cypress" />
+/// <reference path="../support/index.d.ts" />
+
+// Use `cy.dataCy` custom command for more robust tests
+// See https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements
+
+describe("Landing", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+  it(".should() - assert that <title> is correct", () => {
+    cy.title().should("include", "Quasar");
+  });
+});
+
+// describe('Home page tests', () => {
```

Comment:
Generally a good idea to either remove commented code or include why it is commented out and when it should be uncommented or deleted.  Otherwise it tends live forever without purpose.

Reply:
This was here before I start changing stuff around (it's marked green because I changed it's folder).
I think it was meant to show how to write some tests with Cypress.
Will add a comment about this

Reply:
I added some comments on why that code is there